docs/inputs.rst

```diff
@@ -165,7 +165,8 @@ parenthesis). As such, they are all optional settings, although some of them (su
 size) often need to be set by the user for proper operation. The list of possible options (with any 
 default value provided in parentheses) is:
 
- - dtM (0.001) – desired mooring model time step (s)
+ - dtM (3.402823e+38) – desired mooring model maximum time step (s)
+ - CFL (0.5) – desired mooring model maximum CFL factor
```

Comment:
These CFL notes are helpful, but need to be in the V2 section (this is V1). Additionally they should be under the line "The following MoorDyn-C options are not supported by MoorDyn-F:"

Reply:
Fixed on https://github.com/FloatingArrayDesign/MoorDyn/pull/182/commits/5c3a404c3b2ee7809ebda94754e611c4c4f30691